README.md

```diff
@@ -4,6 +4,23 @@
 
 This project aims to make it easier to integrate secure code signing into a CI pipeline by using cloud-based hardware security module(HSM)-protected keys. This project is part of the [.NET Foundation](https://www.dotnetfoundation.org/) and operates under their [code of conduct](https://www.dotnetfoundation.org/code-of-conduct). It is licensed under [MIT](https://opensource.org/licenses/MIT) (an OSI approved license).
 
+## Prerequisites
+
+- An up-to-date x64-based version of Windows currently in [mainstream support](https://learn.microsoft.com/lifecycle/products/)
+- [.NET 8 SDK or later](https://dotnet.microsoft.com/download)
+- [Microsoft Visual C++ 14 runtime](https://aka.ms/vs/17/release/vc_redist.x64.exe)
+
+## Install
+
+To install Sign CLI in the current directory, open a command prompt, path to the directory where Sign CLI will be installed, and execute:
+
+```
+dotnet new tool-manifest
+dotnet tool install --local --prerelease sign
+```
+
+To run Sign CLI, execute `dotnet sign` from the same directory.
```

Comment:
this should just be `sign.exe` if installed with the `--tool-path .` parameter above.